app/controllers/application_controller.rb

```diff
@@ -11,4 +11,14 @@ def require_user
     render file: '/public/401' unless current_user
   end
 
+  def hour_min(runtime)
+    hours = runtime / 60
+    rest = runtime % 60
+    "#{hours} hr #{rest} min"
+  end
```

Comment:
I wonder if these will eventually live in the service and or the poro we create?